pymeasure/instruments/agilent/agilentL4534A.py

```diff
@@ -0,0 +1,333 @@
+    )
+
+    def clear_display(self):
+        return self.write('DISP:TEXT:CLE')
+
+    arm_source = Instrument.control(
+        "CONF:ARM:SOUR?",
+        "CONF:ARM:SOUR %s",
+        """Set the source used to arm the digitizer (IMMediate|EXTernal|SOFTware|TIMer)""",
+        validator=strict_discrete_set,
+        values=['IMM', 'SOFT', 'EXT', 'TIM']
+    )
+
+    ext_slope = Instrument.control(
+        "CONF:EXT:INP?",
+        "CONF:EXT:INP %s",
+        """Set the edge to be used for the external trigger input, NEGative or POSitive""",
+        validator=strict_discrete_set,
+        values=['NEG', 'POS']
+    )
+
+    TRIGGER_SOURCE_VALUES = ['IMM', 'SOFT', 'EXT', 'CHAN', 'OR']
+
+    trigger_source = Instrument.control(
+        "CONF:TRIG:SOUR?",
+        "CONF:TRIG:SOUR %s",
+        """Set the trigger source {IMMediate|SOFTware|EXTernal|CHANnel|OR}""",
+        validator=strict_discrete_set,
+        values=TRIGGER_SOURCE_VALUES
+    )
+
+    sample_rate = Instrument.control(
+        "CONF:ACQ:SRAT?",
+        "CONF:ACQ:SRAT %d",
+        """Set the sample rate in samples/s (Hz):
+        1000
+        2000
+        5000
+        10000
+        20000
+        50000
+        100000
+        200000
+        500000
+        1000000
+        2000000
+        5000000
+        10000000
+        20000000
+        """,
+        validator=strict_discrete_set,
+        values=SAMPLE_RATE_VALUES,
+        set_process=lambda v: v.m_as(ureg.Hz),  # send the value as Hz to the device
+        get_process=lambda v: ureg.Quantity(v, ureg.Hz)  # convert to quantity
+    )
+
+    samples_per_record = Instrument.control(
+        "CONF:ACQ:SCO?",
+        "CONF:ACQ:SCO %d",
+        """Control number of samples that will be captured for each trigger, in multiples of 4 (8-128,000,000)""",
+        validator=sample_count_function,
+        values=SAMPLE_COUNT_VALUES
+    )
+
+    pre_trigger_samples = Instrument.control(
+        "CONF:ACQ:SPR?",
+        "CONF:ACQ:SPR %d",
+        """Control number of samples that will captured pre-trigger, in multiples of 4 (8-128,000,000)""",
+        validator=sample_count_function,
+        values=PRE_TRIG_VALUES
+    )
+
+    number_of_records = Instrument.control(
+        "CONF:ACQ:REC?",
+        "CONF:ACQ:REQ %d",
+        """Control number of of records that will be captured before arming is disabled (1-1024)""",
+        validator=strict_range,
+        values=RECORD_COUNT_VALUES
+    )
+
+    trigger_holdoff = Instrument.control(
+        "CONF:ACQ:THOL?",
+        "CONF:ACQ:THOL %e",
+        """Control minimum time between triggers, during which trigger inputs are ignored,
+          in decimal seconds (0-10s)""",
+        validator=strict_range,
+        values=TRIGGER_HOLDOFF_VALUES
+    )
+
+    trigger_delay = Instrument.control(
+        "CONF:ACQ:TDEL?",
+        "CONF:ACQ:TDEL %e",
+        """Control delay time after trigger before acquistion starts, in decimal seconds (0-3600s)""",
```

Comment:
What are "decimal seconds"?

Reply:
Seconds expressed as a decimal, e.g. 1.5 or 0.01, so not just whole seconds. Maybe there's a clearer way to phrase that?

Reply:
In our documentation (https://pymeasure.readthedocs.io/en/latest/dev/coding_standards.html#docstrings) we used the example to state the type in parenthesis: "Control something (float).".